README.md

```diff
@@ -173,5 +173,6 @@ has been passionate about computer science since he was a 13-year-old, his first
 #### Errata
 * Page 82 (Dropping databases, first paragraph): **to drop a table** _should be_ **to drop a database**
 * Page 581 (Section 5, heading): **The PostegreSQL System** _should be_ **The PostgreSQL System**
+* Page 114 (Using the IN/NOT IN condition, last paragraph): **the values `pk=2` and `pk=4`** _should be_ **the single value `pk=11`**; **that have `pk=2` or `pk=4`** _should be_ **that have `category=11`**.
```

Comment:
I think this could be even less clear, as it is written. Probably we should mention the whole statement, that is 
```This statement extracts the values pk=2 and pk=4 from the category table and the external
query searches the records in the posts table that have pk=2 or pk=4.```
should be substitued with
```This statement extracts the value that corresponds to 'organe' from the category table and the external
query searches the records in the posts table that have such category, resulting in the tuples with pk=2 or pk=4